togetherly/lib/providers/chore_provider.dart

```diff
@@ -146,7 +170,7 @@ class ChoreProvider with ChangeNotifier {
     if (familyId != null) {
       _allChores = await _choreService.getChoresByFamily(familyId);
       _allAssignments =
-          await _assignmentService.getAssignmentsByFamily(familyId);
+      await _assignmentService.getAssignmentsByFamily(familyId);
```

Comment:
This does not seem like the correct formatting. Perhaps you need to run "dart format" again?